template.yaml

```diff
@@ -0,0 +1,35 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Echelon-Backend
+
+  SAM Template for Echelon-Backend
+  
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 3
+
+Resources:
+  AuthenticationFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: API/authentication/
+      Handler: app.lambdaHandler
+      Runtime: nodejs10.x
```

Comment:
Did we want to keep it as nodejs10?

Reply:
Yes, given that Node.js v8 hits EOL in December, and v10 is the current active LTS. If we have issues we can look at v8 or maybe even v12.